vboxlauncher@mockturtl/files/vboxlauncher@mockturtl/applet.js

```diff
@@ -133,11 +141,11 @@ MyApplet.prototype = {
       this.addErrorMessage(_("ERROR. No compatible virtual machine programs found."))
   }
 
-, launcherFor: function(title, cmd, callback) {
+, launcherFor: function(title, cmd, callback, callbackHeadless) {
```

Comment:
Is there a typo here?  This function is only called in L135, L136, and neither callsite passes the new argument.

Reply:
Now that I look at it, everything named "launcher" is really a menuItem.  We should change names to reflect that.

Reply:
I remove the callbackHeadless as this method is only called to display the menuItem for Vbox and VMware.

Those launchers open the corresponding apps.
How would you like them to be called ?

Reply:
It's fine for now.